activerecord/lib/active_record/errors.rb

```diff
@@ -1,6 +1,47 @@
 module ActiveRecord
-  # = Active Record Errors
+  # = Active \Record \Errors
   #
+  # Subclasses <tt>ActiveModel::Errors</tt> class
+  # to implement copying association errors
+  # to the foreign key
+  class Errors < ActiveModel::Errors
```

Comment:
Maybe something along the lines of:

> Subclass of `ActiveModel::Errors` that properly includes association errors
> on the foreign key.

may be better. What do you think ? And could you wrap these documentation lines around 80 chars please ?